docs/scos/dev/example-modules.md

```diff
@@ -0,0 +1,66 @@
+
+## The difference between core and example modules
+
+**Core modules** provide generic, reusable functionality that benefits a wide range of projects. They offer a foundation for customization and serve as versatile building blocks.
+
+**Example Modules** in contrast, are designed for specific, one-off use-cases. Their functionality is tightly aligned with unique business requirements and isn't suitable for broad reuse. These modules serve as specialized, project-specific solutions.
+
+## Example module use-cases
+
+### SCOS: Showcase functionality
+
+SCOS uses example modules to fill the gap with concrete/example implementation over an abstract workflow (latter is the target of delivery), where the concrete/example implementation is never actually usable by any concrete customer or business use-case, thus the actual business logic is not part of the core (eg: Example picking strategy).
+
+### SCOS: Unique business use-case
+
+SCOS uses example modules to implement functionality that is unique business (customer) use-case specific. The actual business logic is not part of the core as it does NOT offer regular core quality and is extracted into an example module.
+
+## Obligations
```

Comment:
- This title is too legal.
- Some items are internal guidelines only.
- Some items are too direct.

Reply:
True, yes, I should have caught that... The next commit will address this.

Reply:
Updated